website/src/components/footer/Footer.jsx

```diff
@@ -14,16 +14,82 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React from 'react';
+import React from "react";
 
-import "./footer.css"
+import "./footer.css";
+import logo from "../../assets/logo.png";
+import {
+  AiFillGithub,
+  AiFillSlackCircle,
+  AiFillTwitterCircle,
+} from "react-icons/ai";
+import { BsStackOverflow } from "react-icons/bs";
 
 const Footer = () => {
-    return (
-        <div className='footer justify--content'>
-            <span>Copyright © 2023</span>
+  const icons = [
+    { url: "https://github.com/vitessio/arewefastyet", icon: AiFillGithub },
+    { url: "https://vitess.io/slack", icon: AiFillSlackCircle },
+    { url: "https://twitter.com/vitessio", icon: AiFillTwitterCircle },
+    {
+      url: "https://stackoverflow.com/search?q=vitess",
+      icon: BsStackOverflow,
+    },
```

Comment:
Let's inline these fours lines too, just like we did in lines `30`, `31` and `32`

Reply:
okay